integrations/crowdnode/src/main/java/org/dash/wallet/integrations/crowdnode/ui/portal/TransferFragment.kt

```diff
@@ -62,7 +62,7 @@ class TransferFragment : Fragment(R.layout.fragment_transfer) {
     private var balanceAnimator: ObjectAnimator? = null
 
     @Inject
-    lateinit var securityModel: SecurityModel
+    lateinit var ISecurityFunctions: ISecurityFunctions
```

Comment:
Was `securityModel` supposed to be renamed as `ISecurityModel` or was it a Find/Release without being case sensitive?

Reply:
Yeah that is the case, thanks for noticing